specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -2156,6 +2195,72 @@
         "password"
       ]
     },
+    "SSISAccessCredential": {
+      "description": "SSIS access credential.",
+      "type":"object",
+      "properties": {
+        "domain": {
+          "type": "object",
+          "description": "Domain for windows authentication."
+        },
+        "userName": {
+          "type": "object",
```

Comment:
strings ?

Reply:
We support AKV for the two properties, so the type is object.